crates/astria-conductor/src/sequencer/builder.rs

```diff
@@ -10,6 +10,7 @@ pub(crate) struct Builder {
     pub(crate) sequencer_grpc_client: SequencerGrpcClient,
     pub(crate) sequencer_cometbft_client: sequencer_client::HttpClient,
     pub(crate) sequencer_block_time: Duration,
+    pub(crate) sequencer_chain_id: String,
```

Comment:
Since this is a builder, this would be nicer:

```suggestion
    pub(crate) expected_sequencer_chain_id: String,
```